### ISSUE TITLE
[Bug] (Maybe) incorrect embeddings calculation?

### ISSUE THREAD

**Issue by langchain4j** [xenova/transformers.js#296]
Hi,

It seems that embeddings produced by transformers.js are a bit different from those calculated by transformers python lib.

Your example:
```js
let extractor = await pipeline('feature-extraction', 'Xenova/all-MiniLM-L6-v2');
let result = await extractor('This is a simple test.', { pooling: 'mean', normalize: true });
console.log(result);
// Tensor {
//     type: 'float32',
//     data: Float32Array [0.09094982594251633, -0.014774246141314507, ...],
//     dims: [1, 384]
// }
```

If we use the same model and pooling/normalization algo in python, we'll get a bit different values (code taken from https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2):
```python
import torch
import torch.nn.functional as F
from transformers import AutoTokenizer, AutoModel


def mean_pooling(model_output, attention_mask):
    token_embeddings = model_output[0]
    input_mask_expanded = attention_mask.unsqueeze(-1).expand(token_embeddings.size()).float()
    return torch.sum(token_embeddings * input_mask_expanded, 1) / torch.clamp(input_mask_expanded.sum(1), min=1e-9)


sentences = ['This is a simple test.']

tokenizer = AutoTokenizer.from_pretrained('sentence-transformers/all-MiniLM-L6-v2')
model = AutoModel.from_pretrained('sentence-transformers/all-MiniLM-L6-v2')

encoded_input = tokenizer(sentences, padding=True, truncation=True, return_tensors='pt')

with torch.no_grad():
    model_output = model(**encoded_input)

sentence_embeddings = mean_pooling(model_output, encoded_input['attention_mask'])

sentence_embeddings = F.normalize(sentence_embeddings, p=2, dim=1)

print(sentence_embeddings)
# tensor([[ 9.5282e-02, -2.4730e-02,  8.3197e-03,  3.6906e-02, -1.0667e-02, ...
```

**How to reproduce**
```js
import * as transformers from 'https://cdn.jsdelivr.net/npm/@xenova/transformers@2.6.0';

let extractor = await transformers.pipeline('feature-extraction', 'Xenova/all-MiniLM-L6-v2');
let output = await extractor('This is a simple test.', { pooling: 'mean', normalize: true });
console.log(output.data)
// Float32Array(384) [0.08325746655464172, -0.029848560690879822, 0.011567391455173492, 0.04427928104996681, -0.003810760099440813, -0.05299655348062515, -0.05639844387769699, 0.01874053291976452, 0.01376376673579216, ...

```
**Expected behavior**
Embeddings produced by transformers.js should have the same or very similar values to those produced by transformers python lib.

**Environment**
- Transformers.js version: 2.6.0
- Browser (if applicable): Chrome
- Operating system (if applicable): Windows

**Comment by xenova** [comment]
Hi there! 👋  To get the exact same output, you can use the unquantized version of the model:
```js
let extractor = await transformers.pipeline('feature-extraction', 'Xenova/all-MiniLM-L6-v2', { quantized: false });
```

The reason for using the quantized model first was a design decision made a long time ago when the library was only for in-browser usage (and so we opted for the massive reduction in memory)
